proto/engine/v1/epbs.proto

```diff
@@ -0,0 +1,122 @@
+// Copyright 2024 Prysmatic Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+package ethereum.engine.v1;
+
+import "proto/eth/ext/options.proto";
+import "proto/engine/v1/execution_engine.proto";
+
+option csharp_namespace = "Ethereum.Engine.V1";
+option go_package = "github.com/prysmaticlabs/prysm/v4/proto/engine/v1;enginev1";
+option java_multiple_files = true;
+option java_outer_classname = "ExecutionEngineProto";
+option java_package = "org.ethereum.engine.v1";
+option php_namespace = "Ethereum\\Engine\\v1";
+
+
+message PayloadAttestationData {
+	bytes beacon_block_root = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+	uint64 slot 		= 2 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.Slot"];
+	bool payload_present 	= 3;
+}
+
+message PayloadAttestation {
+  	bytes aggregation_bits          = 1 [(ethereum.eth.ext.ssz_size) = "ptc.size", (ethereum.eth.ext.cast_type) = "ptc.type"];
+	PayloadAttestationData data 	= 2;
+    	bytes signature 		= 3 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message PayloadAttestationMessage {
+	uint64 validator_index 		= 1 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.ValidatorIndex"];
+	PayloadAttestationData data 	= 2;
+    	bytes signature 		= 3 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message InclusionListSummaryEntry {
+	bytes address 	 = 1 [(ethereum.eth.ext.ssz_size) = "20"];
+	uint64 gas_limit = 2;
+}
+
+message InclusionListSummary {
+	uint64 proposer_index 		  	   = 1 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.ValidatorIndex"];
+	repeated InclusionListSummaryEntry entries = 2 [(ethereum.eth.ext.ssz_max) = "max_inclusion_list.size"];
+}
+
+message SignedInclusionListSummary {
+	InclusionListSummary message = 1;
+    	bytes signature 	     = 2 [(ethereum.eth.ext.ssz_size) = "96"];
+}
+
+message InclusionList {
+	SignedInclusionListSummary summary = 1;
+	uint64 slot 			   = 2 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.Slot"];
+	repeated bytes transactions 	   = 3 [(ethereum.eth.ext.ssz_size) = "?,?", (ethereum.eth.ext.ssz_max)  = "max_inclusion_list.size,1073741824"];
```

Comment:
Irrelevant to your PR but I wish we have a config value for `1073741824`

Reply:
right, but that's how we have transactions currently

Reply:
I can make a PR against develop for this on the actual payload proto